qdata/parser.py

```diff
@@ -287,16 +385,32 @@ def uop(self, *args):
         return Gate(op_name, params, wires)
 
     def anylist(self, *args):
-        # either <idlist> or <mixedlist>
+        """Either ``idlist`` or ``mixedlist``"""
         return ParsedList(Lists.ANYLIST, unpack(args))
 
     def idlist(self, *args):
+        """Either a single id, or a list of id's. Note that
+        an id is an alphanumeric label (with underscores allowed)
+        that *must* begin with a lowercase character."""
```

Comment:
Let's drop this, it's completely unnecessary

Reply:
Do you mean the docstring, or the lowercase character restriction 😆

Reply:
lowercase :)